### DISCUSSION TITLE
哈希算法，迭代计算问题求教

### DISCUSSION THREAD

**Discussion by songjinlian** [Leon406/ToolsFx#220] [Q&A] [answered]
测试数据如下，当times>1时，输出结果不相同了，不知道这情况情况是正常的还是不正常的？
<html xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns="http://www.w3.org/TR/REC-html40">
<head>

<meta name=Generator content="Microsoft Excel">

<style>

</style>
</head>
<body>

待处理数据 | 待处理数据格式 | 哈希算法 | times=1 | times=2 | times=3
-- | -- | -- | -- | -- | --
A | raw | MD5 | 7fc56270e7a70fa81a5935b72eacbe29 | 4cbd6d53280de25e04712c7434a70642 | 01484f32e1eae8c021a3bd2fb3d1062f
QQ== | base64 | MD5 | 7fc56270e7a70fa81a5935b72eacbe29 | 226c5b52c6281f39ffab2da3cf30254e | a834e103452df36ae1adcd7540725531
41 | hex | MD5 | 7fc56270e7a70fa81a5935b72eacbe29 | 8f28f2e7231860115d2a8cacba019dbe | 385ebb54f2384e60c347dc7a0c304b4d

</body>

</html>

  
待处理数据	待处理数据格式	哈希算法	times=1	times=2	times=3
A	raw	MD5	7fc56270e7a70fa81a5935b72eacbe29	4cbd6d53280de25e04712c7434a70642	01484f32e1eae8c021a3bd2fb3d1062f
QQ==	base64	MD5	7fc56270e7a70fa81a5935b72eacbe29	226c5b52c6281f39ffab2da3cf30254e	a834e103452df36ae1adcd7540725531
41	hex	MD5	7fc56270e7a70fa81a5935b72eacbe29	8f28f2e7231860115d2a8cacba019dbe	385ebb54f2384e60c347dc7a0c304b4d

**Comment by Leon406** [comment, marked as answer]
https://github.com/Leon406/ToolsFx/commit/d0e3670158f19c5dd64e99bc29f6c72c96156d60
https://github.com/Leon406/ToolsFx/actions/runs/7782175297
